### ISSUE TITLE
npmjs.com not searchable

### ISSUE THREAD

**Issue by yubo9807** [luncheon/esbuild-plugin-jsx-html-entry#1]
![image](https://user-images.githubusercontent.com/95408465/226241111-2709dc65-2cc5-4d14-9fd1-96b79e8da652.png)

**Comment by luncheon** [comment]
Yes, this is my personal experimental toy and I have not published it to npm. I am honored that you opened this issue as I thought no one would use it. Thanks.

You can use this plugin in several ways.

- You can install by `npm i -D luncheon/esbuild-plugin-jsx-html-entry` from GitHub.
- You are free to clone it, modify it, and publish it on npm as yours.
    - Since the source code for this plugin is available under the [WTFPL](http://www.wtfpl.net/).
- You can copy the [source code](https://github.com/luncheon/esbuild-plugin-jsx-html-entry/blob/main/index.js) and pass it to esbuild without publishing it.
